functions/error-reporting.ts

```diff
@@ -24,6 +25,15 @@ export const handler: Handler = async ({ body }) => {
       user,
     } = JSON.parse(body)
     Bugsnag.notify({ name, message, stack, cause }, (event) => {
+      event.app = {
+        releaseStage: 'production',
+        version: cliVersion,
```

Comment:
we are reporting it twice now, should we remove it from device?

Reply:
This is required to have it here: 
![CleanShot 2023-04-19 at 16 57 31](https://user-images.githubusercontent.com/11156362/233116283-1ae53ab5-6dc4-4d9f-9f38-860237e1823f.png)


But I will remove the runtime version does not seemed to get picked up

Reply:
The node version should apparently be added automatically, according to the docs.